vm/vm.go

```diff
@@ -1128,6 +1141,46 @@ func (vm *VM) loadAcceptedBlocks(ctx context.Context) error {
 	return nil
 }
 
+func (vm *VM) restoreAcceptedQueue(ctx context.Context) error {
+	has, err := vm.HasLastProcessed()
+	if err != nil {
+		return errors.New("could not determine if have last processed")
+	}
```

Comment:
nit: this should wrap the original error with `fmt.Errorf(...)`

Reply:
Done thanks!